com.ibm.streamsx.topology/com.ibm.streamsx.topology.functional.python/CCWindow/CCWindow_h.cgt

```diff
@@ -46,6 +46,7 @@ if ($pyoutstyle eq 'dict') {
     // Members
     // Control for interaction with Python
     SplpyFuncOp *funcop_;
+    SplpyFuncOp *spl_in_object_out;

```

Comment:
I know this is a WIP but using two `SplpyFuncOp` instances is dangerous because it's more than a function pointer. The class represents an operator instance that is a functional operator, not just the function pointer. It would be better to either have an explicit PyObject* that is the second function pointer or extend `SplpyFuncOp` to represent operator with with two functions.

Reply:
Using a `SplpyFuncOp` has been restricted to wrapping user provided functions, of which there is only one for CCWindow. In the future, if there are two user provided functions for an operator, `SplpyFuncOp` could be extended to allow wrapping multiple user functions.